vms/vms/static/vms/js/hide-organization.js

```diff
@@ -0,0 +1,10 @@
+/** Hides textbox for unlisted organization if a listed organization is selected */
+function hideOrganization(){
+  var value = $('#select').val();
+  if (value !== '0') {
+    $("#div_id_unlisted_organization").hide();
```

Comment:
I think you can use `.toggle()` ?

Reply:
@tonythomas01 Implemented the same. Please review.